src/Compute/custom/StrategiesCore/AzureRMAsyncCmdlet.cs

```diff
@@ -0,0 +1,54 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+
+using Microsoft.Azure.Commands.ResourceManager.Common;
+using Microsoft.Azure.PowerShell.Cmdlets.Compute.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Text;
+using System.Threading;
+
+namespace Microsoft.Azure.PowerShell.Cmdlets.Compute.Strategies
+{
+    public class AzureRMAsyncCmdlet : AzureRMCmdlet, IStrategiesCmdletAdapter
+    {
+        //[ArgumentCompleter(typeof(SubscriptionArgumentCompleter))]
```

Comment:
@markcowl we should remove this comment if we aren't going to use the `SubscriptionArgumentCompleter`